### PR TITLE
Add z-stream approver for sdn related repo

### PR DESCRIPTION
**Pull request by zhaozhanqi**
Based on this [Designating QE Z-Stream Cherry-Pick Approv](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit#heading=h.p2kb03i1f8l)ers for a Repository , I submit this PR.

@openshift/team-sdn-qe please help review in case I missed one, thanks.

### CODE REVIEW

File: core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ label:
     - allowed_users:
       - rioliu-rh
       - sergiordlr
+      - anuragthehatter
+      - rbbratta
+      - zhaozhanqi
+      - huiran0826
```
Comment (asood-rh) [inline core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml:18]: @zhaozhanqi Is there a reason for this list does not have all the team members?

Comment (sinnykumari) [reply core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml:21]: Agree with @zhaozhanqi , I was wondering if can we keep this list minimal(maybe 1 or 2 members)? MCO has code owned by few more teams and this list will keep growing and would be difficult to keep track.

Comment (rioliu-rh) [reply core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml:21]: suggest to add 2 members per team (1 backup)

**Comment by sdodson** [comment]
@zhaozhanqi need to address the test failures before this will merge.

File: core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml (hunk 1 above)
Comment (zhaozhanqi) [reply core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml]: I updated with two sdn members

Comment (sinnykumari) [reply core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml]: Thanks @zhaozhanqi !

**Review by sinnykumari — APPROVED** [review]

**Pull request merged**